core/primitives/CMakeLists.txt

```diff
@@ -5,22 +5,17 @@
 
 add_library(primitives
```

Comment:
remove this lib, as it does not contain cpps

Reply:
actually it does contain 2 cpps, and later only one will remain.